tests/ha/ha_install.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Install HA
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+use base 'sles4sap';
+use testapi;
+use utils;
+use serial_terminal 'select_serial_terminal';
+use power_action_utils 'power_action';
+use strict;
+use warnings;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    # Workaround for current agama installation: registration, extensions and so on
+    # Install suseconnect-ng
+    zypper_call('in suseconnect-ng');
+    # De-register SLES
+    assert_script_run('suseconnect -d');
+    # Regster SLES
+    assert_script_run('SUSEConnect' . ' -r ' . get_required_var('SCC_REGCODE') . ' --url ' . get_required_var('SCC_URL'));
```

Comment:
This part is done in system_prepare which is implemented in #21018

Reply:
Agreed. Seems changes in `ha_install` are not required now that #21018 has been merged. Can you rebase and retry the VRs dropping this module?

Reply:
Rebased, dropped ha_install.pm, added workaround for HA extension as https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21018 did not register HA extension.